scenes/game/local_game.gd

```diff
@@ -477,6 +479,7 @@ class Player:
 	var do_not_cleanup_buddy_this_turn : bool
 	var cannot_move : bool
 	var cannot_move_past_opponent : bool
+	var cannot_move_past_opponent_buddy_id : Variant
```

Comment:
why bother putting Variant?

Reply:
i guess since it is assumed string maybe but this is nullable.

Reply:
yeah, I wanted to avoid making two variables for this; maybe that would be clearer?

Reply:
it's fine i guess